com.unity.multiplayer.mlapi/Tests/Runtime/RPCQueueTests.cs

```diff
@@ -0,0 +1,110 @@
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+using MLAPI.SceneManagement;
+using MLAPI.Transports.UNET;
+
+namespace MLAPI.RuntimeTests
+{
+    public class RPCQueueTests
+    {
+        private NetworkingManager m_NetMan;
+
+        [UnitySetUp]
+        public IEnumerator SetUp()
+        {
+            yield return new EnterPlayMode();
+        }
+
+        /// <summary>
+        /// RPCQueueUnitTest
+        /// Tests the egress and ingress RPC queue functionality
+        /// ** This does not include any of the MLAPI to Transport code **
+        /// </summary>
+        /// <returns>IEnumerator</returns>
+        [UnityTest]
+        public IEnumerator RPCQueueUnitTest()
+        {
+            var NetManObject = new GameObject();
+            m_NetMan = NetManObject.AddComponent<NetworkingManager>();
+            UnetTransport unetTransport = NetManObject.AddComponent<UnetTransport>();
```

Comment:
FWIW, here's how I stood up a NetworkManager
```suggestion
         GameObject o = new GameObject();
        NetworkingManager nm = (NetworkingManager)o.AddComponent(typeof(NetworkingManager));
        nm.SetSingleton();
        nm.NetworkConfig = new NetworkConfig();
        UnetTransport ut = (UnetTransport)o.AddComponent(typeof(UnetTransport));

        ut.ServerListenPort = 7777;
        nm.NetworkConfig.NetworkTransport = ut;

        byte CustomChannel = 0;

        // test 1: add a legit channel.
        ut.Channels.Add(new UnetChannel() {Id = Channel.ChannelUnused + CustomChannel, Type = QosType.Unreliable});

        try
        {
            nm.StartServer();
        }
        catch (InvalidChannelException e)
        {
            Assert.Fail("The UNet transport won't allow registration of a legit user channel");
        }

        nm.StopServer();
        nm.Shutdown();
```

Reply:
Well the spinning down part I could add, but the typeof aspect is the same as the templatized version.  As a bonus, the templatized version uses less characters.  :)